testimony/constants.py

```diff
@@ -7,18 +7,9 @@
 CLR_ERR = 'red'
 CLR_GOOD = 'green'
 
-DOC_TAGS = [
```

Comment:
The tags are now parsed inside the TestCase class, no need to define the tags here since now other parts of the code use the instance attributes instead of dictionary items.